presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcTypeHandle.java

```diff
@@ -29,20 +29,23 @@
     private final int columnSize;
     private final int decimalDigits;
     private final Optional<Integer> arrayDimensions;
+    private final Optional<CaseSensitivity> caseSensitivity;
 
     @JsonCreator
     public JdbcTypeHandle(
             @JsonProperty("jdbcType") int jdbcType,
             @JsonProperty("jdbcTypeName") Optional<String> jdbcTypeName,
             @JsonProperty("columnSize") int columnSize,
             @JsonProperty("decimalDigits") int decimalDigits,
-            @JsonProperty("arrayDimensions") Optional<Integer> arrayDimensions)
+            @JsonProperty("arrayDimensions") Optional<Integer> arrayDimensions,
+            @JsonProperty("caseSensitivity") Optional<CaseSensitivity> caseSensitivity)
```

Comment:
Restore ctor without `Optional<CaseSensitivity>` for backwards compatibility.

We can mark it deprecated and remove it later, but I would rather go with a builder approach (later).
For now let's just have two (non-deprecated) constructors, because it's not clear which one
should be retained (if any).